crates/sui-adapter-transactional-tests/tests/child_count/count_decremented.exp

```diff
@@ -31,16 +31,17 @@ created: object(111), object(112)
 written: object(110)
 
 task 7 'view-object'. lines 78-78:
-Owner: Account Address ( A )
+Owner: Object ID: ( fake(111) )
```

Comment:
@emmazzz do the changes in this file make sense?

Reply:
how come this test is broken?

Reply:
Hmm iiuc updating narwhal pointers shouldn't affect this test. @tnowacki since he recently added this test and made changes to child count.

Reply:
Another possibility is that this is caused by the `rand 0.7.3 -> 0.8.5` upgrade. It seems unlikely this is a new bug caused by the `rand` and `Narwhal` upgrades.